lms/views/lti/basic_launch.py

```diff
@@ -94,6 +81,15 @@ def unconfigured_launch(self):
 
         This happens if we cannot resolve a document URL for any reason.
         """
+
+        if not self.request.has_permission(Permissions.LTI_CONFIGURE_ASSIGNMENT):
+            # Looks like the user is not an instructor, so show an error page
+
+            # https://docs.pylonsproject.org/projects/pyramid/en/latest/narr
+            # /renderers.html?highlight=override_renderer#overriding-a-renderer-at-runtime
+            self.request.override_renderer = "lms:templates/lti/basic_launch/unconfigured_launch_not_authorized.html.jinja2"
```

Comment:
I was trying to find that. We could do that, but I think it executes the render right away. As mentioned here:

https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/introduction.html?highlight=render_to_response#write-testable-views

> When you use a [renderer](https://docs.pylonsproject.org/projects/pyramid/en/latest/glossary.html#term-renderer) with your view callable, you are freed from needing to return a "webby" Response object. Instead your views can return a simple Python dictionary. Pyramid will take care of rendering the information in that dictionary to a Response on your behalf. **As a result, your views are more easily tested, since you don't need to parse HTML to evaluate the results.**

I think we would have to parse the results of the HTML to check what we got. I don't think we want to go that far, certainly not now?